### ISSUE TITLE
Customer.getCashBalanceTransactions does not exist

### ISSUE THREAD

**Issue by SamueleAlpino** [stripe/stripe-java#1761]
### Describe the bug

I am currently utilizing stripe-java version 24.18.0 and adhering to the guidelines provided in the Stripe documentation (https://docs.stripe.com/api/customer_balance_transactions/list) in order to retrieve a list of customer balance transactions. However, I have encountered an issue wherein the code I copied fails to compile due to the absence of the getCashBalanceTransactions method within the customer class. I am curious whether this discrepancy may be attributed to an oversight in the documentation, as it references a method that seems to be unimplemented.

### To Reproduce

As docs:
```
Stripe.apiKey = {api_key};
Customer customer = Customer.retrieve("customer_id");
CustomerCashBalanceTransactionCollectionListParams params =
 CustomerCashBalanceTransactionCollectionListParams.builder().setLimit(3L).build();
CustomerCashBalanceTransactionCollection customerCashBalanceTransactions =
  customer.getCashBalanceTransactions().list(params);
```

getCashBalanceTransactions() does not exist

### Expected behavior

I expect that one of the latest versions (24.18.0) includes the method used in the documentation

### OS

macOS

### Java version

Java 17

### stripe-java version

v24.18.0 

### API version

2023-10-16

**Comment by helenye-stripe** [comment]
Hi @SamueleAlpino! Thanks for bringing this to our attention. We’re working on getting this fixed.

In the meantime, you can use our new `StripeClient` pattern to retrieve this as a workaround, more information is available in our [README](https://github.com/stripe/stripe-java?tab=readme-ov-file#usage). Here is an example for listing `CustomerCashBalanceTransactions`. :
```java
StripeClient client = new StripeClient("sk_test_...");

CustomerCashBalanceTransactionCollectionListParams params =
 CustomerCashBalanceTransactionCollectionListParams.builder().setLimit(3L).build();
client.customers().cashBalanceTransactions().list(params)
```

**Comment by helenye-stripe** [comment]
This has been fixed as of the release v24.20.0. Updated docs are also available at https://docs.stripe.com/api/cash_balance_transactions/list?lang=java.
